blocks/http/test/qa_HttpKnownBlocks.cpp

```diff
@@ -0,0 +1,22 @@
+#include <fmt/format.h>
+#include <fmt/ranges.h>
+
+#include <cassert>
+
+#include <gnuradio-4.0/http/all.hpp>
+
+int
+main() {
+    using namespace std::string_literals;
+
+    auto known = gr::globalBlockRegistry().knownBlocks();
+    std::ranges::sort(known);
+
+    std::vector<std::string> desired{
+        //
+        "gr::http::HttpBlock"s, //
+    };
+    std::ranges::sort(desired);
+
+    assert((std::ranges::includes(known, desired)));
```

Comment:
brittle test depends on sorting. Intersection might be better. Also, this test is nearly identical to others and should probably be grouped to one for simplicity.

Reply:
See the earlier comment.

I'd like to group them, but they are in different components.